frontend/src/components/accounts/AccountDetails.tsx

```diff
@@ -25,12 +25,20 @@ export default class extends React.Component<Props> {
               title="Ⓝ Balance"
               text={<Balance amount={account.amount} />}
               className="border-0"
+              href={"https://docs.nearprotocol.com/docs/concepts/account"}
```

Comment:
1. Looking at this interface I would expect that we define the whole CardCell to become a link when I pass `href`, but it is just a link to the docs in the popup... I think it does not make a lot of sense to make this separation and it makes things easier to customize if you just explicitly include `<a href...>docs</a>` in the `termDescription`.

2. `href` and `termDesription` do not belong to `CardCell`. Let's not abuse it, unless it is absolutely necessary. To me, it makes a lot of sense to just have a standalone component `TermHelpButton`, and you just use it in the title:

    ```jsx
            <CardCell
              title={
                <>
                  {"Ⓝ Balance "}
                  <TermHelpButton>
                    {"Total NEAR token balance. This includes the 'Locked' balance. "}
                    <a href="https://docs.nearprotocol.com/docs/concepts/account">docs</a>
                  </TermHelpButton>
                </>
              }
              text={<Balance amount={account.amount} />}
              className="border-0"
            />
    ```